src/config/dynamic-theme-fixes.config

```diff
@@ -2347,6 +2347,118 @@ CSS
 
 ================================
 
+tweakers.net
+
+CSS
+.ankeiler>a.commentCount {
+    background-image: url(../../../g/icons/commentcounter_small_bg.svg) !important;
+    text-decoration: none !important;
+}
+.thumb.category {
+   border: none !important;
+}
+#categoryBrowser li a{
+     background: none !important;
```

Comment:
Use 4 spaces

Reply:
Fixed Dev tools Doesn't Support tab? (It just unfocus the page)